plugin/al/plugin/AL_USDMayaTestPlugin/AL/usdmaya/commands/test_ProxyShapeSelect.cpp

```diff
@@ -107,27 +164,33 @@ TEST(ProxyShapeSelect, selectNode1)
     MSelectionList sl;
 
     EXPECT_EQ(1u, results.length());
-    EXPECT_EQ(MString("|transform1|root|hip1|knee1|ankle1|ltoe1"), results[0]);
+    if (!MayaUsdProxyShapePlugin::useVP2_NativeUSD_Rendering()) {
+        EXPECT_EQ(MString("|transform1|root|hip1|knee1|ankle1|ltoe1"), results[0]);
+    } else {
+        EXPECT_EQ(
+            MString("|world|transform1|AL_usdmaya_ProxyShape1/root/hip1/knee1/ankle1/ltoe1"),
```

Comment:
I never like to see the |world UFE path component in a path literal, as it is arguably a UFE implementation detail --- should use Ufe::PathString::string() (and the reverse, Ufe::PathString::path()) wherever possible, which will remove this component.